### DISCUSSION TITLE
Best way to style buttons on hover?

### DISCUSSION THREAD

**Discussion by jeremy-deutsch** [microsoft/react-native-windows#8645] [Q&A] [answered]
Building a simple RNW app, there are a couple things I want to be able to do, but can't find any documentation for:
- Applying different styles to a button/view on mouse hover (e.g. color, opacity, size)
- Changing the mouse cursor to a pointer while hovering a button/view

Is there an official, stable way to handle these cases? I'd imagine Microsoft has some way they're handling them in their apps.

**Comment by chrisglein** [comment, marked as answer]
There's a [big thread here](https://github.com/microsoft/react-native-windows/issues/2099) about an effort to make the hover states (and other mouse-centric things) more unified across all platforms. There are options to do what you want in RNW today, but they're underdocumented (for example compared to [the keyboard props](https://microsoft.github.io/react-native-windows/docs/ikeyboardprops-api)). The lack of documentation on these props is [tracked here](https://github.com/microsoft/react-native-windows-samples/issues/358).

There are some running examples in this repo, [like this](https://github.com/microsoft/react-native-windows/blob/003602b90aeda1b2a43702c6a9307c0f01010e59/packages/%40react-native-windows/tester/src/js/examples-win/Mouse/MouseExample.windows.js). But I don't see anything proper [on the samples repo](https://github.com/microsoft/react-native-windows-samples/search?q=onMouseEnter). I've opened [an issue](https://github.com/microsoft/react-native-windows-samples/issues/541) to track the lack of a sample that covers this.

@rectified95 FYI
